python/src/ops.cpp

```diff
@@ -1316,6 +1316,30 @@ void init_ops(py::module_& m) {
         Returns:
             array: The output array with the specified shape and values.
       )pbdoc");
+  m.def(
```

Comment:
This does not build on my machine. When I execute ``pip install -e .`` I get the following error:
```
      /Users/gboduljak/Desktop/mlx-franc/mlx/python/src/ops.cpp:1328:18: error: use of undeclared identifier 'Scatter'
            "mode"_a = Scatter::ReduceType::None,
```
This is because we did not import ``Scatter``. I will fix this by exposing the mode as a string. We will map the string to the appropriate enum value.

Reply:
@francescofarina I am now working on this.

Reply:
@francescofarina Please take a look at https://github.com/francescofarina/mlx/pull/1.